app/views/my/reminders.html.erb

```diff
@@ -0,0 +1,40 @@
+<%#-- copyright
+OpenProject is an open source project management software.
+Copyright (C) 2012-2024 the OpenProject GmbH
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2013 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See COPYRIGHT and LICENSE files for more details.
+
+++#%>
+
+<% html_title(t(:label_my_account), I18n.t("js.reminders.settings.title")) -%>
+
+<%=
+  render(Primer::OpenProject::PageHeader.new) do |header|
+    header.with_title { I18n.t("js.reminders.settings.title") }
+    header.with_breadcrumbs([{ href: my_account_path, text: t(:label_my_account) },
+                             I18n.t("js.reminders.settings.title")])
+  end
+%>
+
+<%= angular_component_tag "opce-reminder-settings", inputs: { show_toolbar: false } %>
```

Comment:
The `show_toolbar` flag was introduced, because the myPage still needed the toolbar whereas the administration settings already had the PageHeade in place. So we can now get rid of the flag and remove the toolbar altogether from the component